app/routes/__index/domains/new.tsx

```diff
@@ -45,16 +46,20 @@ export const action = async ({ request }: ActionArgs) => {
   // the domain name (e.g., `foo` in `foo.username.root.com`).
   const { data } = newDnsRecordParams;
 
-  // Create a pending record... for now
-  // This will most likely be replaced by some other logic
-  await addDnsRequest({
-    username: user.username,
-    type: data.type,
-    name: data.name,
-    value: data.value,
-  });
+  try {
+    await addDnsRequest({
+      username: user.username,
+      type: data.type,
+      subdomain: data.name,
+      value: data.value,
+    });
 
-  return redirect(`/domains`);
+    return redirect(`/domains`);
+  } catch (error) {
+    logger.warn('Add DNS request error', error);
+    //Need to display an error response
```

Comment:
Please file that as an issue to fix later.

Reply:
I wrote #364